hub/core/transform/test_transform.py

```diff
@@ -6,10 +7,14 @@
 from hub.util.remove_cache import remove_memory_cache
 from hub.tests.common import parametrize_num_workers
 from hub.tests.dataset_fixtures import enabled_datasets
-from hub.util.exceptions import InvalidOutputDatasetError
+from hub.util.exceptions import InvalidOutputDatasetError, TransformError
 
 all_compressions = pytest.mark.parametrize("sample_compression", [None, "png", "jpeg"])
 
+schedulers = ["threaded", "processed"]
+schedulers = schedulers + ["ray"] if ray_installed() else schedulers
+all_schedulers = pytest.mark.parametrize("scheduler", schedulers)
```

Comment:
can we not omit tests like this? we should always parametrize with ray. we should skip ray tests by default, but when `--ray` is provided with pytest it should fail if not installed.